flink/src/test/java/org/apache/iceberg/flink/sink/TestIcebergFilesCommitter.java

```diff
@@ -528,7 +528,8 @@ private static TestOperatorFactory of(String tablePath) {
     @Override
     @SuppressWarnings("unchecked")
     public <T extends StreamOperator<Void>> T createStreamOperator(StreamOperatorParameters<Void> param) {
-      IcebergFilesCommitter committer = new IcebergFilesCommitter(TableLoader.fromHadoopTable(tablePath), CONF, false);
+      IcebergFilesCommitter committer = new IcebergFilesCommitter(TableLoader.fromHadoopTable(tablePath), CONF, false,
```

Comment:
We will need more unit tests in this operator test,  addressing that the `IcebergFilesCommitter` will work as expected exactly.

Reply:
BTW, do we need sql test for the disabled checkpoint case ?

Reply:
You are right, I will add more UTs